.travis.yml

```diff
@@ -24,7 +24,7 @@ script:
   - cd docs; make html
   # Run the test suit with coverage
   - cd ..
-  - coverage run --source=axelrod -m unittest discover
+  - travis_wait 30 coverage run --source=axelrod -m unittest discover
```

Comment:
If travis receives no output for a default 10 minutes then it stops and returns a failure. This changes the default to 30 mins but also changes how it runs it (https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received) - that's why I tested that failures were still being reported correctly.

Even with the test trimming the 10 minutes default was still causing failures on travis (I think due to travis being a bit slow and some hypothesis tests - I ran numerous experiments and wasn't getting this locally) so this just ensures that it shouldn't happen.